{"version":3,"file":"ModelNotFoundException.js","sourceRoot":"","sources":["../../../src/api/exceptions/ModelNotFoundException.ts"],"names":[],"mappings":";AAAA,yDAAyD;AACzD,mEAAmE;AACnE,iFAAiF;;AAEjF;;;;;;;GAOG;AAEH,wDAAqD;AAGrD,4BAAoC,SAAQ,qBAAS;IACjD,YAAY,IAAY;QACpB,KAAK,CAAC,GAAG,EAAE,yBAAyB,IAAI,iBAAiB,CAAC,CAAC;IAC/D,CAAC;CACJ;AAJD,wDAIC","sourcesContent":["// Copyright (c) 2017-2018, The Particl Market developers\n// Distributed under the GPL software license, see the accompanying\n// file COPYING or https://github.com/particl/particl-market/blob/develop/LICENSE\n\n/**\n * MessageException\n * ----------------------------------------\n *\n * This should be used if a someone requests a\n * entity with a id, but there is no entity with this id in the\n * database, then we throw this exception.\n */\n\nimport { Exception } from '../../core/api/Exception';\n\n\nexport class ModelNotFoundException extends Exception {\n    constructor(name: string) {\n        super(404, `Model with identifier ${name} does not exist`);\n    }\n}\n"]}